sources/packages/api/src/route-controllers/application/application.controller.ts

```diff
@@ -131,8 +145,92 @@ export class ApplicationController extends BaseController {
         "Error while associating the assessment workflow.",
       );
     }
+  }
+
+  /**
+   * Creates a new application draft for the authenticated student.
+   * The student is allowed to have only one draft application, so
+   * this method will create the draft or throw an exception in case
+   * of the draft already exists.
+   * @param payload payload to create the draft application.
+   * @param userToken token from the authenticated student.
+   * @returns the application id of the created draft or an
+   * HTTP exception if it is not possible to create it.
+   */
+  @AllowAuthorizedParty(AuthorizedParties.student)
+  @Post("draft")
+  async createDraft(
```

Comment:
PLease rename it to createDraftApplication

Reply:
Done!